scripts/invalidate_cloudfront_cache.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Set DISTRIBUTION_ID to the ID of your CloudFront distribution
+DISTRIBUTION_ID=$(aws cloudfront list-distributions --query "DistributionList.Items[].Id" --output text)
+
+# Invalidate all objects in the distribution
+aws cloudfront create-invalidation --distribution-id "$DISTRIBUTION_ID" --paths "/*"
+
+echo "CloudFront cache invalidation started for distribution $DISTRIBUTION_ID"
```

Comment:
Do you think it's worth capturing the invalidation ID as a response, and waiting for the invalidation to be complete before returning?

(i.e. using something like [aws cloudfront wait](https://docs.aws.amazon.com/cli/latest/reference/cloudfront/wait/invalidation-completed.html)?)